src/makeRobot.js

```diff
@@ -38,7 +38,57 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return 'name: ' + this.name + ', chip version: '
+      + this.version + ', wheels: ' + this.wheels;
+    },
+    get location() {
+      return this.name + ': x=' + this.coords.x + ', y=' + this.coords.y;
+    },
+    goForward(steps = 1) {
+      if (steps > 0) {
+        this.coords.y += steps;
+      }
+
+      return this;
+    },
+    goBack(steps = 1) {
```

Comment:
```suggestion
    get info() {
      return 'name: ' + this.name + ', chip version: '
      + this.version + ', wheels: ' + this.wheels;
    },
    
    get location() {
      return this.name + ': x=' + this.coords.x + ', y=' + this.coords.y;
    },
    
    goForward(steps = 1) {
      if (steps > 0) {
        this.coords.y += steps;
      }

      return this;
    },
    
    goBack(steps = 1) {
```

Add empty line between all your methods